### PR TITLE
[#24] - Configure frontend theme

### LINKED ISSUE #24: Frontend Theme and Font
**Is your feature request related to a problem? Please describe.**
Once we have decided on a colour palette and font to use on the front end, these colours should be added to a centralized style file so that they can be imported into any style sheet. This should ensure consistent colouring on all components and will also make it easy to change theme in the future.

**Describe the solution you'd like**
Example file using default Material UI colours:

image

Example usage:

image

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by nzbasic**
# Description:
We have decided on a colour palette for our app, this PR adds these colours to our theme.scss file, and also configures an MUI ThemeProvider so that our MUI components will follow the same theme.

Closes #24 

### All Submissions:

* [x] Have you followed the guidelines in our [Contributing document](../CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] My code follows the code style of this project
- [ ] My code has been commented
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes
- [ ] All new and existing tests passed

### CODE REVIEW

File: frontend/src/App.jsx [hunk 1]
```diff
@@ -1,9 +1,29 @@
...
+    },
+  },
+});
+
 const App = () => (
     <div className={classes.container}>
-      <Router />
+      <ThemeProvider theme={theme}>
+        <Router />
+      </ThemeProvider>
```
Comment (Michaelrecycle) [inline frontend/src/App.jsx]: Should theme provider be at the top level? Doesn't really matter but feels like a more comfortable hierarchy

**Review by Michaelrecycle — COMMENTED** [review]
Nicely done

File: frontend/src/App.jsx (hunk 1 above)
Comment (nzbasic) [reply frontend/src/App.jsx]: You are right, I have moved it to be the root component

**Review by Michaelrecycle — APPROVED** [review]
beautiful

**Pull request merged**